src/TileDBClient/TileDBClient.ts

```diff
@@ -24,7 +24,7 @@ const isNode = typeof process === "object";
 
 if (isNode) {
   if (process.env.TILEDB_REST_HOST) {
-    defaultConfig.basePath = process.env.TILEDB_REST_HOST;
+    defaultConfig.basePath = process.env.TILEDB_REST_HOST + "/v1";
```

Comment:
@SarantopoulosKon some route are `/v2`, such as query. Should this be hardcoded to v1?

Reply:
The client class creates the v2 config in its constructor https://github.com/TileDB-Inc/TileDB-Cloud-JS/blob/master/src/TileDBClient/TileDBClient.ts#L52

It looks a bit confusing but thats how the current implementation works 😕 

To me it looks like the client doesn't work at the moment if the TILEDB_REST_HOST is set but does not have the /v1 suffix.
This PR will probably fix the issue, but, do we need to consider this a breaking change?

Reply:
`TILEDB_REST_HOST` is not support to have a suffix. Its idea is the hostname itself so its generic and works for future suffixes. I don't think this is a breaking change, I think the original code was "wrong" so this is a bug fix imo.

Thanks for showing how the v2 client is setup.

Reply:
I believe that besides the fact that this looks confusing, this `str.replace` may also cause problems.
I vote for a small refactor of the config setup:
- No `/v1` hardcoded **anywhere**.
- Both configs should get the host string as-is and append the /v1 and /v2 in the **constructor**.
Nice and clean.

Reply:
@alekshs actually i was thinking about it as well, so let's do it